src/components/KTableView/KTableView.vue

```diff
@@ -240,6 +277,31 @@
                       />
                     </template>
                   </KDropdown>
+
+                  <div
+                    v-else-if="header.key === TableViewHeaderKeys.BULK_ACTIONS"
+                    class="row-bulk-actions"
+                  >
+                    <KCheckbox
+                      v-model="row.selected"
+                      :aria-label="header.label"
+                      class="bulk-actions-checkbox"
+                      data-testid="bulk-actions-checkbox"
+                      :disabled="!getRowBulkActionEnabled(row)"
+                    />
+
+                    <KTooltip
+                      v-if="!getRowBulkActionEnabled(row) && getRowBulkActionTooltip(row)"
+                      :text="getRowBulkActionTooltip(row)"
+                    >
+                      <InfoIcon
+                        class="bulk-actions-tooltip-trigger"
+                        :color="`var(--kui-color-text-neutral, ${KUI_COLOR_TEXT_NEUTRAL})`"
+                        :size="KUI_ICON_SIZE_30"
+                        tabindex="0"
+                      />
+                    </KTooltip>
```

Comment:
issue: we should just show the disabled tooltip when hovering over the disabled checkbox; we don't need another icon in the table

Reply:
Done